src/core/client/admin/routes/Moderate/ModerateNavigation/Navigation.tsx

```diff
@@ -69,22 +71,24 @@ const Navigation: FunctionComponent<Props> = ({
 
   return (
     <SubBarNavigation>
-      <NavigationLink to={moderationLinks[0]}>
-        <Icon>flag</Icon>
-        <Localized id="moderate-navigation-reported">
-          <span>Reported</span>
-        </Localized>
-        {isNumber(reportedCount) && (
-          <Counter data-testid="moderate-navigation-reported-count">
-            <Localized
-              id="moderate-navigation-comment-count"
-              $count={reportedCount}
-            >
-              {reportedCount}
-            </Localized>
-          </Counter>
-        )}
-      </NavigationLink>
+      {mode && mode === "POST" && (
```

Comment:
Technically you can just do `mode === "POST"` instead of checking `mode && mode === "POST"`

Reply:
We should actually modify this logic to instead be:

```suggestion
      {(mode === "POST" || (isNumber(reportedCount) && reportedCount > 0) && (
```

Because even if the moderation mode is set to "PRE", we still could have comments that are reported. With this logic, it'll hide itself only if there is no reported comments and is not in "POST" moderation.